### ISSUE TITLE
How to contribute?

### ISSUE THREAD

**Issue by mirdaki** [10leej/tordath#1]
I think this is an interesting idea! I was wondering how to best set expectations for contributing. In the sense of:
- What things can be added (cults, nations, events, historical characters)?
- What existing things can be changed or removed (if X nation is too similar to Y, should they be merged to prevent confusion)? And what should a discussion around that look like?
- What is set in stone and shouldn't change (if anything)?
- What is the tone/goal of this world (realistic low magic, out of this world crazy action, etc)? That would help others have a north star of sorts to guide them and be a good way to decide if something doesn't fit well in the world (aliens with flying saucers abducting adventures might be cool, but jarring).

I don't expect full answers to theses questions (I like the idea of this being sort of an experiment), but I'd like to get an idea of what you're thinking about (particularly the tone of the world one)

**Comment by 10leej** [comment]
# Not that I'm slow or uncaring, I just haven't looked at github

> What things can be added (cults, nations, events, historical characters)?

Feel free to add, that said I more or less have plenty of nations and organizations listed but no real detail behind them.

> What existing things can be changed or removed (if X nation is too similar to Y, should they be merged to prevent confusion)? And what should a discussion around that look like?

Generally add remove whatever you want in a pull request, as long as the discussion itself.

> What is set in stone and shouldn't change (if anything)?

So the way i have it envisioned is that Tor'Dath is a singular continent as of this time, it's basically at a pre industrial revolution era  so trying to keep it around there would be ideal.

> What is the tone/goal of this world (realistic low magic, out of this world crazy action, etc)? That would help others have a north star of sorts to guide them and be a good way to decide if something doesn't fit well in the world (aliens with flying saucers abducting adventures might be cool, but jarring).

So, it's steampunk. Well not steampunk, maybe... aetherpunk? I'm basically making my own [Kaladesh](https://media.wizards.com/2017/downloads/magic/Plane-Shift_Kaladesh.pdf) but it's just "not quite there" in terms of how magic items have proliferated in daily life.  
It's basically right at the dawn of an industrial revolution, and some areas still use swords, other places figured out how to put a BFG on a motorized carriage.  
There's trains, and airships, but still plenty of horses pulling around carts for travelling merchants.  
The big thing, is that it's also sitting right on the edge of it's "first" world war. So working out how the different nations interact with each other in terms of trade/milirary needs to be fairly important. I kinda hint at the idea of it being basically [Limbuld](https://github.com/10leej/tordath/blob/master/Nations/The-Kingdom-of-Limbuld.md) vs [Taran](https://github.com/10leej/tordath/blob/master/Nations/The-Taran-Empire.md).  

I do have a campaign currently coming up on it's end so really I should be investing more into this world a small bit, but in general you got a cool idea. Just make a pull request and we'll discuss it. It's still very much an early project, so I don't really have much idea on anything that's not written. And stuff that is, (especially if there's not much there to begin with) I'm willing to let it be changed.  
Once the worlds fleshed out enough I might setup branches based on time lines so we could have "future tordath" that's gone full final fantasy high magic, or "past tordath" that's not really all that different from how most people view most campaign settings.  

## TL:DR  
Make a pull request and find out. The worst I can say is no, and grammar nazi's are appreciated since I only just barely passed my language arts classes 10 years ago.

**Comment by mirdaki** [comment]
Thanks for the guidance! I've made a PR expanding a bit about Pilto #2
